here-naksha-app-service/src/test/java/com/here/naksha/app/service/ReadFeaturesByIdsTest.java

```diff
@@ -221,4 +218,114 @@ void tc0407_testReadFeaturesWithCommaSeparatedIds() throws Exception {
     ;
   }
 
+  @Test
+  void tc0408_withPropSelection() throws Exception {
+    // Test API : GET /hub/spaces/{spaceId}/features
+    // Validate features getting returned for Ids provided, in which only selected properties are returned
+    // Given: Features By Ids request (against existing space)
+    final String idsQueryParam = "id=my-custom-id-400-1,my-custom-id-400-2";
+    final String streamId = UUID.randomUUID().toString();
+    final String expectedBodyPart =
+            loadFileOrFail("ReadFeatures/ByIds/TC0408_withPropSelection/feature_response_part.json")
+                    .replaceAll("\\{\\{streamId}}",streamId);
+    final String selectionParams = "selection=p.speedLimit,%s".formatted(urlEncoded("p.@ns:com:here:xyz.tags"));
```

Comment:
Same comment as pervious one. How can we assure that "selection" is doing the job if features doesn't have any additional properties than what we are selecting?? Pls add some attributes to original features